### ISSUE TITLE
Reset  Function Don't work

### ISSUE THREAD

**Issue by ahmadjehad** [jordanopensource/portal-web#360]
Reset Password at https://members.jordanopensource.org/forgot-password Don't work

Best Regards.

**Comment by mahasneh** [comment]
This is true indeed, but it's specific to the Members Hub, whose code is hosted in another repo. I will open an issue there. Thanks!

**Comment by ahmadjehad** [comment]
i wanted to register didnt work because "my email exist already" so i tried to reset it and that didnt work too

**Comment by mahasneh** [comment]
Hello Ahmad, can you try to see if the reset email went to the spam/junk folder? Thanks!

**Comment by ahmadjehad** [comment]
i did , nothing is there
